lib/const.rb

```diff
@@ -70,20 +69,25 @@
 CREW_SHRINK_ARCHIVE = ENV['CREW_SHRINK_ARCHIVE']
 
 # Set testing constants from environment variables
-crew_testing_repo = ENV['CREW_TESTING_REPO']
-crew_testing_branch = ENV['CREW_TESTING_BRANCH']
-crew_testing = ENV['CREW_TESTING']
-crew_testing = '0' if crew_testing_repo.nil? || crew_testing_repo.empty?
-crew_testing = '0' if crew_testing_branch.nil? || crew_testing_branch.empty?
-CREW_TESTING = crew_testing
-CREW_TESTING_BRANCH = crew_testing_branch
-CREW_TESTING_REPO = crew_testing_repo
+CREW_TESTING_BRANCH = ENV['CREW_TESTING_BRANCH']
+CREW_TESTING_REPO = ENV['CREW_TESTING_REPO']
+
+CREW_TESTING = unless CREW_TESTING_BRANCH.to_s.empty? or CREW_TESTING_REPO.to_s.empty?
+  ENV['CREW_TESTING']
+else
+  '0'
+end
 
 CREW_USE_PIXZ = ENV['CREW_USE_PIXZ']
 
 USER = `whoami`.chomp
 
-CHROMEOS_RELEASE = `grep CHROMEOS_RELEASE_CHROME_MILESTONE= /etc/lsb-release | cut -d'=' -f2`.chomp
+CHROMEOS_RELEASE = if File.exist?('/etc/lsb-release')
+  File.read('/etc/lsb-release')[/CHROMEOS_RELEASE_CHROME_MILESTONE=(.+)/, 1]
+else
+  # newer version of Chrome OS exports info to env by default
+  ENV['CHROMEOS_RELEASE_CHROME_MILESTONE']
+end
```

Comment:
Turn into an immediate if one-liner?

Reply:
It would be too long if making this in a line

Reply:
How about something like below?
```
CHROMEOS_RELEASE = ( File.exist?('/etc/lsb-release') ) ? \
  File.read('/etc/lsb-release')[/CHROMEOS_RELEASE_CHROME_MILESTONE=(.+)/, 1] : \
  # newer version of Chrome OS exports info to env by default
  ENV['CHROMEOS_RELEASE_CHROME_MILESTONE']
```
Continued one-liner. :)

Reply:
I think if/unless is better then escaping :)